src/index.js

```diff
@@ -3,6 +3,16 @@ import chProcess from 'child_process';
 import _ from 'lodash';
 import fs from 'fs';
 import chokidar from 'chokidar';
+import path from 'path';
+
+let pathSet = false;
+function setPath() {
+  if(!pathSet) {
```

Comment:
Based on how `process.env` works, path will be set for the current process and all child processes.  I'll test out inlining it and make sure that it doesn't repeatedly set it.